test/sql/local/irc/insert/test_basic_insert.test

```diff
@@ -121,11 +121,10 @@ statement ok con1
 commit
 
 # The second commit failed because both transactions tried to add sequence number: 2
-# Invalid Configuration Error: Request to 'http://127.0.0.1:8181/v1/transactions/commit' returned a non-200 status code
-statement error
+statement error con2
 commit
 ----
-TransactionContext Error
+<REGEX>:.*Requirement failed: branch main has changed.*
```

Comment:
Is there a way to check the status code of the HTTP response here? Or in a test of the logs?

Reply:
That falls under logging / retry logic in my opinion
The main goal currently is forwarding the error correctly, we can intercept based on the response code later